### PR TITLE
Release 4.10.2

### PR DESCRIPTION
**Pull request by johnhwhite**
- Fixed cell renderer change when validator status changes. [#143](https://github.com/blackbaud/skyux-ag-grid/pull/143) (Thanks @[blackbaud-jeremymorgan](https://github.com/blackbaud-jeremymorgan)!)

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.10.2 (2021-09-17)
+
+- Fixed cell renderer change when validator status changes. [#143](https://github.com/blackbaud/skyux-ag-grid/pull/143) (Thanks @[blackbaud-jeremymorgan](https://github.com/blackbaud-jeremymorgan)!)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: I feel like we're missing some words here. I'm not sure, but here's my best guess of what we mean:
```suggestion
- Fixed the cell renderer to refresh when the validator status changes. [#143](https://github.com/blackbaud/skyux-ag-grid/pull/143) (Thanks @[blackbaud-jeremymorgan](https://github.com/blackbaud-jeremymorgan)!)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2)

**Review by blackbaud-johnly — DISMISSED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.2)

**Pull request merged**